cinema_service/settings.py

```diff
@@ -134,7 +143,7 @@
 STATIC_URL = "static/"
 
 MEDIA_URL = "/media/"
-MEDIA_ROOT = BASE_DIR / "media"
+MEDIA_ROOT = "/files/media"
```

Comment:
The `MEDIA_ROOT` path is set to `/files/media`. Ensure this directory exists and is correctly configured to store media files.